lib/foreman_puppet.rb

```diff
@@ -1,5 +1,6 @@
 module ForemanPuppet
   FOREMAN_EXTRACTION_VERSION = '2.6'.freeze
```

Comment:
I think in the 1.0 branch, you have a commit changing this to 3.0, probably should be aligned in master too ;)

Reply:
it should, but given we want to drop it ASAP from master, I think it's ok not to :)